tests/test_generate_ics.py

```
@@ -0,0 +1,330 @@
+
+
+class TestGenerateIcs(TestCase):
+    @staticmethod
+    @contextmanager
+    def argv(*args):
+        with patch("sys.argv", ["generate_ics.py", *args]):
+            yield
+
+    @staticmethod
+    @contextmanager
+    def stderr():
```

Ruff (0.15.15)

[warning] Missing return type annotation for staticmethod `stderr`

(ANN205)